src/main/resources/standardTestCases/non_root_default_user/non_root_default_user.wdl

```diff
@@ -0,0 +1,21 @@
+task notroot {
+  command {
+    echo $HOME
+  }
+
+  runtime {
+    docker: "mcovarr/notroot:v1"
```

Comment:
Yeah @geoffjentry and I were discussing this yesterday, it would be good to do the same for the Centaur Docker as well (see `centaur.wdl` in Cromwell).  I tried to push my image to the Broad repo yesterday but was denied access; do you know how to do this?